GalaxyZooTouchTable/GalaxyZooTouchTable/ViewModels/SpaceViewModel.cs

```diff
@@ -0,0 +1,61 @@
+﻿using GalaxyZooTouchTable.Models;
+using PanoptesNetClient;
+using PanoptesNetClient.Models;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.Threading.Tasks;
+
+namespace GalaxyZooTouchTable.ViewModels
+{
+    public class SpaceViewModel : ViewModelBase
+    {
+        public double RA { get; set; } = 250.3035;
+        public double DEC { get; set; } = 35.09;
+        public double SCALE { get; set; } = 1.5;
```

Comment:
Are these the hardcoded values you were referring to in the description of the PR?